draft-ietf-taps-interface.md

```diff
@@ -2380,11 +2383,27 @@ Default:
 
 This property specifies that a message should be sent and received as a single
 packet without network-layer fragmentation, if possible.
-Attempts to send a message with this property set to a size greater than the
+This only takes effect when used with transports supporting this functionality
+and running over IP version 4. When it does take effect, setting this property to
+true will cause the Don't Fragment bit to be set in the IP header, and
+attempts to send a message with this property set to a size greater than the
 transport's current estimate of its maximum packet size (`singularTransmissionMsgMaxLen`)
-will
-result in a `SendError`. When used with transports supporting this functionality
-and running over IP version 4, the Don't Fragment bit will be set.
+will result in a `SendError`.
+
+#### No Transport-Layer Fragmentation {#no-transport-fragmentation}
+
+Name:
+: noTransportFragmentation
+
+Type:
+: Boolean
+
+Default:
+: false
+
+This property specifies that a message should be sent and received as a single
+network-layer datagram without transport-layer fragmentation, if possible.
+This only takes effect when used with transports supporting this functionality.
 
```

Comment:
I suggest something like: 

When set to true, this property requests the network layer at the 
sending endpoint to not fragment the packets generated by the
transport layer.  When running over IPv4, setting this property to
true will also cause the Don't Fragment bit to be set in the IP header
When this property is set, an
attempt to send a message size greater than the
transport's current estimate of its maximum packet size (`singularTransmissionMsgMaxLen`)
will result in a `SendError`.
This only takes effect when the transport and network layer 
support this functionality.

Reply:
Thanks!  I now used this text; because it says very clearly what is going on, it seemed ok to me to also call the heading "No Segmentation".